packages/pubsub/lib/utils/serde.ts

```diff
@@ -0,0 +1,23 @@
+import { serialize, deserialize } from 'v8';
```

Comment:
not using json because stringify/parse are not symetrical operations.
Since encoding/decoding is all happening internally I though zod validation would add a lot of overhead with all the schemas. 
I am happy to use something more portable like messagePack if you prefer.